ospf6d/ospf6_zebra.c

```diff
@@ -328,7 +328,7 @@ static void ospf6_zebra_route_update(int type, struct ospf6_route *request,
 	else
 		ret = zclient_route_send(ZEBRA_ROUTE_ADD, zclient, &api);
 
-	if (ret < 0)
```

Comment:
is `ret` used for other things, or should its type be updated ?

Reply:
fixed in a later commit.